shared/router-v2/account-switcher/index.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react'
+import * as Kb from '../../common-adapters'
+import * as Styles from '../../styles'
+import HeaderHOC from '../../common-adapters/header-hoc'
+import * as ConfigTypes from '../../constants/types/config'
+import {Props as HeaderHocProps} from '../../common-adapters/header-hoc/types'
+
+export type AccountRowItem = {
+  account: ConfigTypes.ConfiguredAccount
+  fullName: string
+}
+export type RowsProps = {}
```

Comment:
why?

Reply:
I guess each thing got removed from this one-by-one and then it got refactored out, and I forgot to remove it.